test-specifications/cryptoService/accountCreateTransaction.md

```diff
@@ -0,0 +1,155 @@
+# AccountCreateTransaction - Test specification
+
+## Description:
+This test specification for AccountCreateTransaction is to be one of many for testing the functionality of the Hedera SDKs. The SDK under test will use the language specific JSON-RPC server return responses back to the test driver.
+
+## Design:
+Each test within the test specification is linked to one of the properties within AccountCreateTransaction. Each property is tested with a mix of boundaries. The inputs for each test are a range of valid, minimum, maximum, negative and invalid values for the method. The expected response of a passed test can be a correct error response code or seen as the result of node queries. Success on the consensus node can be obtained by a queries such as AccountInfoQuery or AccountBalanceQuery, and on the mirror node through the rest API. Error codes are obtained from the response code proto files.
```

Comment:
When you say success on consensus node what do you mean? Like to successfully see the account property was modified. Or do you mean success as in the transaction was successful? I was thinking success as in getting the record or receipt of the transaction.

Reply:
I updated the language here to (hopefully) be a bit more clear.